runWorkflowXToYggHbb.py

```diff
@@ -44,6 +44,9 @@ def modelNonResBkg(doFailedFits, nonResYears, masses, nonResBkgTrees, procTempla
       command = ''
       for m in masses:
         proc = procTemplate+m
+        # Need to remove output files for the doFailedFits option to work on repeat runs
+        if not doFailedFits:
```

Comment:
Not needed, as we are already in an `if` clause that requires that `doFailedFits` to be `false` (cf. line 42). Please remove it, and I merge.